### ISSUE TITLE
FeedbackChallenge : Liqwid Protocol Security Audit

### ISSUE THREAD

**Issue by stephen-rowan** [Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal#68]
## FeedbackChallenge : Liqwid Protocol Security Audit

In the spirit of Project Catalyst's #FeedbackChallenge we are assessing to what extent the Catalyst Funding Process and it's Proposals are **open source** as part of the proposal [Quality-Assurance-DAO](https://cardano.ideascale.com/a/dtd/Quality-Assurance-DAO/351678-48088). Your proposal has been selected because "open source" is mentioned in it's text or comments.

We would appreciate it if you could respond to the following questions (in the reply to my comments below) : 

1.  **Open** - Is your Organisation open-source ?

> 1. Yes our protocol codebase will be made open source under Apache 2.0 with the 1st components being made open source as early as our testnet app launch end of April/early May.

2.  **Accessible** -  Can community contributors issue pull requests or raise issues on your source code ?

> 2. Currently as our team is building Plutus contracts on a rapidly updating codebase we have made our repos private so they are not currently accessible to the community. Our developers are working with Haskell experts at Well-Typed and the protocol will be audited by Tweag in July, as mentioned above the first Plutus contracts released in the testnet app will be made open source and our team will gradually open up more of the codebase leading up to Plutus mainnet launch.

3.  **Transparent** - Can community contributors access and propose changes to the source text of your documentation ?

> 3. We can include the WP document on our GitHub to allow for community feedback yes.

To keep the assessment transparent each comment is recorded as a [GitHub Issue]( https://github.com/Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal/issues/68 ) on **Quality-Assurance-DAO's** proposal repository (Apache-2.0 License) where a more detailed explanation can be found. 

Together we can build a community of open source contributors on the Cardano platform.

Thank You 

Stephen Whitenstall, QA-DAO

## Detailed explanation 

Your [Issue reference](https://github.com/Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal/issues/68) includes a more detailed explanation of the methodology of this FeedbackChallenge : Open Source. There is also a *Open Source Assessment* [Project Board](https://github.com/Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal/projects/3) and a GitHub bot post Issues in my channel #quality-assurance-dao on Project Catalyst's Discord.

We believe this is important because many developers are concerned with the lack of a clear commitment to open source and fear their free contributions will be locked into a proprietary platform.

## Open Source Assessment Method
- [x] Draft a basic Open Source Assessment Method.
https://github.com/Quality-Assurance-DAO/DAO-Open-Source/tree/main/Documents/Assessments

### Basic Open Source Assessment
1.  **Open** - Is your Organisation open-source ?
1.  **Accessible** -  Can your users issue pull requests or raise issues on your organisation's source code ?
1.  **Transparent** -  Can your users access and propose changes to the source of your organisation's documentation ?

### Method
- [x] Do text search of "open source" on Catalyst Projects.
- [x] Post assessment questions in comments of proposal.
- [x] Raise Issue for each proposal's assessment.
- [x] Record response to assessment.
- [ ] Close Issue once proposal submitted and record responses

### Report
- [ ] Collate and publish report of findings with a Creative Commons License

**Comment by stephen-rowan** [comment]
> Liqwid Labs
> Idea Submitter

> swhitenstall Hey Stephen great questions:
> 1. Yes our protocol codebase will be made open source under Apache 2.0 with the 1st components being made open source as early as our testnet app launch end of April/early May.
> 2. Currently as our team is building Plutus contracts on a rapidly updating codebase we have made our repos private so they are not currently accessible to the community. Our developers are working with Haskell experts at Well-Typed and the protocol will be audited by Tweag in July, as mentioned above the first Plutus contracts released in the testnet app will be made open source and our team will gradually open up more of the codebase leading up to Plutus mainnet launch.
> 3. We can include the WP document on our GitHub to allow for community feedback yes.

**Comment by stephen-rowan** [comment]
Hi Liqwid Labs, 

Apologies for the delay in responding.
Thank you for your response. It is very helpful.  I am sending this message by PM because comments are now closed and I have captured your comment at : 

https://github.com/Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal/issues/68

And moved the Issue to "Done". I will close the Open-Source Assessment next Wednesday (27th).
Then I will draft a report and send it to all the commenters for review before publishing. 
Hopefully this will provide a useful overview for the Catalyst open-source community.

Good Luck with your proposal.
